src/GitHub.UI/Converters/IsLastItemInContainerConverter.cs

```diff
@@ -14,8 +14,15 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 			DependencyObject item = (DependencyObject)value;
 			ItemsControl ic = ItemsControl.ItemsControlFromItemContainer(item);
 
-			return ic.ItemContainerGenerator.IndexFromContainer(item) == ic.Items.Count - 1;
-		}
+            if (ic != null && ic.Items != null)
+            {
+                return ic.ItemContainerGenerator.IndexFromContainer(item) == ic.Items.Count - 1;
+            }
+            else
+            {
+                return false;
+            }
+        }
```

Comment:
Indentation seems heavily screwed up here.

Also, this is probably enough:

```
return ic != null ? ic.ItemContainerGenerator.IndexFromContainer(item) == ic.Items.Count - 1 : false;
```

If `ic` is not null than `Items` won't be null either (I peeked, it's created in the getter).